bin/cli.js

```diff
@@ -89,7 +89,7 @@ async function main() {
 
   console.log('Opening', urlToOpen, 'in your default browser')
 
-  await opn(urlToOpen, {wait: false})
+  await open(urlToOpen, {wait: true})
```

Comment:
It's really odd to me that `wait: true` is required to make this work on windows, but I'll just have to take your word for it.

On macOS anyway, `wait: false` is a nicer behaviour. So I think at least on macOS, I'd prefer if we retained `wait: false`. I'm not sure what the right thing do on linux is, but I'd like to conservatively assume that `wait: false` works too. So I think the best change here would be:

```suggestion
  // We'd like to avoid blocking the terminal on the browsing closing,
  // but for some reason this doesn't work at all on Windows if we
  // don't use wait: true.
  const wait = process.platform === "win32";
  await open(urlToOpen, {wait})
```

What do you think?

Reply:
I just checked on my machine (Windows 10) - it seems that just switching from the old `opn` to the new `open` is enough to fix the issue. No need to change the `wait` from what I can tell. Maybe that was only needed in the old version?

Reply:
It doesn't seem to do the trick for me on both my Windows 10 machines. On the other hand, `await` does work if the browser is not already open and causes the script not to exit. I didn't notice that before since I always have Chrome open with a bunch of tabs and in this case speedscope exits without waiting for the browser to close.

I have not been able to test it properly though. `npm run serve` opens speedscope in a different way using a local server which have no issues, and `npm install -g` fails due to trying to run a bash script. So I have in fact been editing the global installation to test things. I might be missing the obvious since I don't develop in node, but I don't know how to properly start the local version in the same way the global installed version does. (In case this makes a difference.)

Reply:
`npm run serve` is indeed totally different. To test the cli in dev, you'll need to run the following in the speedscope repository root:

```
# clone the repo, then...
npm install
npm run build  # This creates the necessary dist/release/index.html files
bin/cli.js
```

Reply:
Oh, my bad then. I didn't realize.

I get `npm ERR! missing script: build` when trying to run `npm run build`. I'll look into it tomorrow.

Reply:
Ah whoops, that's my bad -- try `npm run prepack`.

Reply:
OK

Being a bash script, `npm run prepack` fails on Windows. I ran the commands manually.

So yeah, it only works with `wait: true`. Which is super weird - as it works perfectly well if I use `wait: false` when calling `open` from the node shell.